Artesian/Artesian.SDK/Dto/Auction/AuctionBidValue.cs

```diff
@@ -22,17 +22,54 @@ public AuctionBidValue(double price, double quantity)
         }
 
         /// <summary>
-        /// The Bid Price
+        /// The Auction Data for a single Multiple timestamps
+        /// </summary>
+        /// <param name="price"></param>
+        /// <param name="quantity"></param>
+        /// <param name="acceptedPrice"></param>
+        /// <param name="acceptedQuantity"></param>
+        /// <param name="blockType"></param>
+        public AuctionBidValue(double price, double quantity, double? acceptedPrice, double? acceptedQuantity, BlockType? blockType = (BlockType?)null)
```

Comment:
the cast should not be needed

also accept* should default to null for retrocompatibility